### DISCUSSION TITLE
Automation of SQL Server patching

### DISCUSSION THREAD

**Discussion by sharmakneeraj4** [dataplat/dbatools#7422] [General] [answered]
Is there a certain way to patch SQL Server clusters from Dbatool?  When I tried to do it by instance name or by host name it did not work. It simply patched one node of cluster not the other.

**Comment by niphlod** [comment, marked as answer]
no, there isn't. you target servers, not clusters, and you do it according to your business rules ^_^

**Reply by sharmakneeraj4** [reply]
Thanks I will test it out.

**Comment by sharmakneeraj4** [comment]
I tested it for SQL CLusters but it seems like its not working for active active set up. However its working fine for active passive cluster setup. Can someone help me on this please?
